Week-4/assignment4.3/fibonacci.js

```diff
@@ -0,0 +1,30 @@
+let fibb = {
+    a : 0,
+    b : 1,
+    pos: 0,
+    [Symbol.iterator](){
+        return this
+    },
+    next(){
+        c = this.a + this.b
```

Comment:
let c or var c, without using let or var, c is created as a global variable

Reply:
Got It